shared/desktop/renderer/index.js

```diff
@@ -145,13 +145,16 @@ const FontLoader = () => (
 
 function render(store, MainComponent) {
   ReactDOM.render(
-    <Root store={store}>
-      <div style={{display: 'flex', flex: 1}}>
-        <RemoteProxies />
-        <FontLoader />
-        <MainComponent />
-      </div>
-    </Root>,
+    // $FlowIssue doesn't understand about StrictMode
+    <React.StrictMode>
```

Comment:
Can we leave this out till we're closer to fully deprecating? It floods errors into the console

Reply:
i removed it in the 2nd pr